go/arrow/cdata/cdata.go

```diff
@@ -433,7 +470,8 @@ func (imp *cimporter) importListLike() error {
 		return err
 	}
 
-	offsets := imp.importOffsetsBuffer(1)
+	offsetSize := imp.dt.Layout().Buffers[1].ByteWidth
```

Comment:
Can the `ByteWidth` trick also be used for String/LargeString instead of having duplicate import routines?